living_documentation_regime/README.md

```diff
@@ -0,0 +1,358 @@
+
+```yaml
+- name: Generate Living Documentation
+  id: generate_living_doc
+  uses: AbsaOSS/living-doc-generator@v0.3.0
+  env:
+    GITHUB-TOKEN: ${{ secrets.LIV_DOC_GENERATOR_ACCESS_TOKEN }}  
+  with:
+    # living documentation regime de/activation
+    liv-doc-regime: true
+    
+    # project verbose (debug) logging feature de/activation
+    verbose-logging: true
+    
+    # output directory path for generated documentation
+    output-path: "/output/directory/path"
+    
+    # input repositories + feature to filter projects
+    liv-doc-repositories: '[
+      {
+        "organization-name": "fin-services",
+        "repository-name": "investment-app",
+        "query-labels": ["feature", "enhancement"],
+        "projects-title-filter": []
+      },
+      {
+        "organization-name": "health-analytics",
+        "repository-name": "patient-data-analysis",
+        "query-labels": ["functionality"],
+        "projects-title-filter": ["Health Data Analysis Project"]
+      },
+      {
+        "organization-name": "open-source-initiative",
+        "repository-name": "community-driven-project",
+        "query-labels": ["improvement"],
+        "projects-title-filter": ["Community Outreach Initiatives", "CDD Project"] 
+      }
+    ]'
+
+    # project state mining feature de/activation
+    liv-doc-project-state-mining: true
+
+    # structured output feature de/activation
+    liv-doc-structured-output: true
+    
+    # group output by topics feature de/activation
+    liv-doc-group-output-by-topics: true
+```
+
+---
+## Regime Configuration
+
+Configure the LivDoc regime by customizing the following parameters based on your needs:
+
+### Inputs
+-**liv-doc-repositories** (optional, `default: '[]'`)
+  - **Description**: A JSON string defining the repositories to be included in the documentation generation.
+  - **Usage**: List each repository with its organization name, repository name, query labels and attached projects you want to filter if any. Only projects with these titles will be considered. For no filtering projects, leave the list empty.
+  - **Example**:
+    ```yaml
+    with:
+      liv-doc-repositories: '[
+      {
+        "organization-name": "fin-services",
+        "repository-name": "investment-app",
+        "query-labels": ["feature", "enhancement"],
+        "projects-title-filter": []
+      },
+      {
+        "organization-name": "health-analytics",
+        "repository-name": "patient-data-analysis",
+        "query-labels": ["functionality"],
+        "projects-title-filter": ["Health Data Analysis Project"]
+      },
+      {
+        "organization-name": "open-source-initiative",
+        "repository-name": "community-driven-project",
+        "query-labels": ["improvement"],
+        "projects-title-filter": ["Community Outreach Initiatives", "CDD Project"] 
+      }
+    ]'
+    ```
+
+### Features De/Activation
+
+- **liv-doc-project-state-mining** (optional, `default: false`)
```

Comment:
Add here link to feature description which is present in chapter below.

Reply:
Good idea, added at: